src/app/components/modals/modal-highlight/modal-highlight.scss

```
@@ -0,0 +1,50 @@
+@import "assets/styles/variables";
+
+.modal-highlight-container {
+  background: #f8f8f8;
+  width: 600px;
+  border-radius: 6px;
+  border: none;
+  outline: none;
+  box-shadow: 0 3px 6px rgba(0, 0, 0, 0.16);
+  padding: 15px;
+}
+.highlight-form {
```

```suggestion
}

.highlight-form {
```
Linebreaks between selectors improve readability